src/App.css

```diff
@@ -1,5 +1,6 @@
 .app {
   text-align: center;
+  background-color: rgba(255, 255, 255, 0.757);
```

Comment:
For simplicity's sake, can we make this a hex? Also, is this white? I checked to see what this is equivalent to and came up with `#ffffff`, in which case I think we can just use `background-color: white`. I could be wrong about this, though...

Reply:
I changed it to like an eggshell white XD